tests/apply.spec.ts

```diff
@@ -1,271 +1,69 @@
 import { test } from '@playwright/test'
 import {
-  ApplyPage,
-  CRNPage,
-  CheckYourAnswersPage,
-  ConfirmPersonPage,
-  ConfirmationPage,
-  DashboardPage,
-  ListPage,
-  StartPage,
-  TasklistPage,
-} from '../pages/apply'
+  checkApplyAnswers,
+  completeAccessCulturalAndHealthcareTask,
+  completeAttachRequiredDocuments,
+  completeBasicInformationTask,
+  completeFurtherConsiderationsTask,
+  completeLocationFactorsTask,
+  completeMoveOnTask,
+  completeOasysImportTask,
+  completePrisonNotesTask,
+  completeRisksAndNeedsTask,
+  completeTypeOfApTask,
+  enterAndConfirmCrn,
+  shouldSeeConfirmationPage,
+  startAnApplication,
+  submitApplication,
+  visitDashboard,
+} from '../steps/apply'
 
-test('test', async ({ page }) => {
-  const dashboard = new DashboardPage(page)
-  await dashboard.goto()
-  await dashboard.clickApply()
+test('Apply for an Approved Premises', async ({ page }) => {
+  // Given I visit the Dashboard
+  const dashboard = await visitDashboard(page)
 
-  const listPage = new ListPage(page)
-  await listPage.startApplication()
+  // And I start an application
```

Comment:
I don't think the comments add much as the methods are very descriptive but they don't take anything away so we might as well leave them